JAICore/jaicore-ml/src/test/java/jaicore/ml/scikitwrapper/ScikitLearn_Wrapper_Test.java

```diff
@@ -41,27 +41,23 @@ public void trainAndTestClassifierRegressionMultitarget() throws Exception {
 		ScikitLearnWrapper slw = new ScikitLearnWrapper("MLPRegressor()", "from sklearn.neural_network import MLPRegressor");
 		Instances datasetTrain = this.loadARFF(test_arff);
 		Instances datasetTest = datasetTrain;
-		int numberInstance = datasetTest.numInstances();
-		slw.setIsRegression(true);
+		slw.setProblemType(ProblemType.REGRESSION);
 		int s = datasetTrain.numAttributes();
 		slw.setTargets(s - 1, s - 2, s - 3);
 		slw.buildClassifier(datasetTrain);
 		double[] result = slw.classifyInstances(datasetTest);
-		// assertNotNull(result);
-		// assertThat(numberInstance * 3, equalTo(result.length));
+		assertEquals("Unequal length of predictions and number of test instances", result.length, datasetTest.size());
 	}
 
 	@Test
 	public void testClassifierRegression() throws Exception {
 		String test_arff = "testsrc/ml/scikitwrapper/Bayesnet_Train.arff";
 		ScikitLearnWrapper slw = new ScikitLearnWrapper("MLPRegressor()", "from sklearn.neural_network import MLPRegressor");
 		Instances datasetTest = this.loadARFF(test_arff);
-		int numberInstance = datasetTest.numInstances();
-		slw.setModelPath(Paths.get("testsrc/ml/scikitwrapper/01673183575_MLPRegressor.pcl").toAbsolutePath().toString());
-		slw.setIsRegression(true);
+		slw.setModelPath(new File("testsrc/ml/scikitwrapper/01673183575_MLPRegressor.pcl").getAbsoluteFile());
```

Comment:
testsrc vs. testrsc? maybe change this here and add "testrsc" as a global resource to not have different testrsc folders for different projects

Reply:
Done